src/evict/evict_inline.h

```diff
@@ -541,6 +544,23 @@ __wt_evict_needed(WT_SESSION_IMPL *session, bool busy, bool readonly, double *pc
     } else {
         dirty_needed = __wt_evict_dirty_needed(session, &pct_dirty);
         updates_needed = __wti_evict_updates_needed(session, &pct_updates);
+
+        /*
+         * Temporary solution to not do updates and dirty eviction using application threads. Log an
+         * error and panic if the cache is full of updates or dirty pages.
+         */
+        if (ignore_updates_dirty && __wt_conn_is_disagg(session) &&
+          !conn->layered_table_manager.leader) {
+            if (pct_updates > pct_full)
+                __wt_verbose_warning(session, WT_VERB_EVICTION, "%s", "cache is full of updates");
+
+            if (pct_dirty > pct_full)
+                __wt_verbose_warning(
+                  session, WT_VERB_EVICTION, "%s", "cache is full of dirty pages");
+
+            WT_STAT_CONN_DSRC_INCR(session, cache_eviction_app_threads_skip_updates_dirty_page);
+            return (false);
```

Comment:
Do we still need to fill in `pct_fullp`? 

Also, should we return `clean_needed` here, so we still do clean eviction of we're over the trigger?

Reply:
Yes, fixed.